app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -19,4 +20,13 @@ def pending_profile_policy
       biometric_comparison_requested: nil,
     )
   end
+
+  def ready_to_verify?
```

Comment:
i like the idea of having this separate method to check the status but the naming doesn't seem quite 100% as in order to be `double_failed` the user would have already gone to the PO and failed.

Reply:
Agreed.

The confusing part here is that this method effectively decides (in `url_for_pending_profile_reason`) whether to take them to the `idv_in_person_ready_to_verify_url` page. I wonder if we actually need to take another step back and look at that method and how we're getting here. 🤔